pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerServiceException.java

```diff
@@ -139,6 +139,12 @@ public TooManyRequestsException(String msg) {
         }
     }
 
+    public static class AlreadyRunningException extends BrokerServiceException {
```

Comment:
I think the exception is a bit ambiguous - what is already running?

Reply:
If you get the exception when trying to run compaction, it's not ambiguous. I left it generic because I thought it could be used for other cases also. I've not problem changing it to CompactionAlreadyRunningException though.